### PR TITLE
Add laptop

### PR DESCRIPTION
**Pull request by Magnus-G**
Rescalable and easy to style laptop for landing pages etc.

image

### CODE REVIEW

**Comment by tysongach** [comment]
@Magnus-G Have you thought about trying to accomplish this with SVG? I think it would be great to avoid all of the empty elements and cut back on all of the CSS.

**Comment by tysongach** [comment]
Quick proof of concept » http://codepen.io/tysongach/pen/qdmbbL?editors=110

File: source/stylesheets/refills/_laptop.scss [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    margin: auto auto -1px;
+    padding: $padding-screen;
+    width: $laptop-lid-width;
+  }
+
+  .laptop-screen {
+    background: url('https://raw.githubusercontent.com/thoughtbot/refills/master/source/images/refills-screenshot2.jpg');
+    background-size: 100%;
+    border-radius: 0.2em;
+    box-shadow: inset 0 0px 1px transparentize(black, 0.2);
```
Comment (jessieay) [inline source/stylesheets/refills/_laptop.scss]: I feel like Hound comments on scss are so long!

**Comment by Magnus-G** [comment]
@tysongach me and @kaishin spoke about this and we both felt like HTML/SCSS would be the best way to go. My idea is that I want to keep as much control as possible in the SCSS and avoid `heights`, `widths`, `colors` and such in the markup. Targeting specific parts of an svg from the SCSS can be tricky as far as I know. In your proof of concept the color of the frame is set in the HTML which makes it hard to align it to other colors we declare in the SCSS.

Does that make sense?

**Comment by tysongach** [comment]
Makes sense. Though, targeting parts of the SVG is trivial, especially for things like colors. It can be done like this:

```css
.frame {
  fill: #f3a057;
}
```

I’m not sure, but the `width` and `height` attributes might be able to be simplified, too. I just didn’t spend much time on that concept.

**Comment by Magnus-G** [comment]
@tysongach I'm still not convinced! But you are welcome to convince me. 

For example: for inset box-shadows and such, would you propose this type of code?

http://www.xanthir.com/b4Yv0

The nice part of an "all HTML" image would be that it's treated the exact same way as the other elements I guess.

**Comment by kaishin** [comment]
At some point we should probably extract those into their own thing. This could go out of hand pretty quickly.

**Comment by Magnus-G** [comment]
@kaishin would you suggest a separate project than Refills for things like this?

**Comment by kaishin** [comment]
@Magnus-G A separate project entirely. There isn't much reuse as far as I see...

**Pull request closed without merging**